site/content/docs/5.0/migration.md

```diff
@@ -7,6 +7,14 @@ aliases: "/migration/"
 toc: true
 ---
 
+## v5.0.0-alpha3
+
+### Sass
+
+- Extended form validation states customization capabilities.
```

Comment:
Could you mention the new arguments for `form-validation-state()` here, please? This would be much clearer and detailed for anyone starting by the migration guide. :)